Userland/Services/AudioServer/main.cpp

```diff
@@ -16,23 +18,33 @@ ErrorOr<int> serenity_main(Main::Arguments)
     TRY(Core::System::pledge("stdio recvfd thread accept cpath rpath wpath unix"));
 
     auto config = TRY(Core::ConfigFile::open_for_app("Audio", Core::ConfigFile::AllowWriting::Yes));
-    TRY(Core::System::unveil(config->filename(), "rwc"sv));
-    TRY(Core::System::unveil("/dev/audio", "wc"));
-    TRY(Core::System::unveil(nullptr, nullptr));
+    // TRY(Core::System::unveil(config->filename(), "rwc"sv));
+    // TRY(Core::System::unveil("/dev/audio", "wc"));
+    // TRY(Core::System::unveil(nullptr, nullptr));
 
     Core::EventLoop event_loop;
     auto mixer = TRY(AudioServer::Mixer::try_create(config));
     auto server = TRY(Core::LocalServer::try_create());
-    TRY(server->take_over_from_system_server());
+    TRY(server->take_over_from_system_server("/tmp/session/%sid/portal/audio"));
 
     server->on_accept = [&](NonnullOwnPtr<Core::LocalSocket> client_socket) {
         static int s_next_client_id = 0;
         int client_id = ++s_next_client_id;
+        dbgln("new normal client {}", client_id);
```

Comment:
Not sure if this is debug output worth keeping; same for the `on_accept` debugging below.